tests/uptane_ci_test.cc

```diff
@@ -0,0 +1,29 @@
+#include <gtest/gtest.h>
+#include <fstream>
+#include <iostream>
+
+#include <logger.h>
+#include <boost/filesystem.hpp>
+
+#include "uptane/uptanerepository.h"
+
+TEST(SotaUptaneClientTest, UptaneSecondaryAdd) {
+  boost::filesystem::remove_all("/tmp/aktualizr");
```

Comment:
We really shouldn't be writing to /tmp. Can we setup the config file to store all temporary files underneath the current build directory?